src/lib/holocene/toaster.svelte

```diff
@@ -0,0 +1,96 @@
+<script lang="ts" context="module">
+  import { Writable, writable } from 'svelte/store';
+  import { v4 } from 'uuid';
+
+  interface Toast {
+    message: string;
+    variant: 'success' | 'error' | 'info' | 'warning' | 'primary';
+    id?: string;
+    duration?: number;
+  }
+
+  interface Toaster {
+    push: (toast: Toast) => void;
+    pop: (id: string) => void;
+    toasts: Writable<Toast[]>;
+  }
+
+  const toasts = writable<Toast[]>([]);
+
+  const push = (toast: Toast) => {
+    const id = toast.id ?? v4();
+    const duration = toast.duration ?? 3000;
+    toasts.update((ts) => [...ts, { id, ...toast }]);
+    const timeoutId = setTimeout(() => {
+      pop(id);
+      clearTimeout(timeoutId);
+    }, duration);
+  };
+
+  const pop = (id: string) => {
+    toasts.update((ts) => ts.filter((t) => t.id !== id));
+  };
+
+  export const toaster: Toaster = {
+    push,
+    pop,
+    toasts,
+  };
+</script>
+
+<script lang="ts">
+  import IconButton from '$holocene/icon-button.svelte';
+  import { crossfade, fly } from 'svelte/transition';
+  import { cubicInOut } from 'svelte/easing';
+  import { flip } from 'svelte/animate';
+
+  export let pop: Toaster['pop'];
+  export let toasts: Toaster['toasts'];
+
+  const [send, receive] = crossfade({
+    duration: 500,
+    easing: cubicInOut,
+    fallback: (node) => fly(node, { x: 250, duration: 250 }),
+  });
+</script>
+
+<div class="absolute top-4 right-4 z-50 flex flex-col items-end gap-2">
```

Comment:
Let's add support for a topRight, topLeft, bottomRight, bottomRight positioning.